test/quantization/pt2e/test_arminductor_quantizer.py

```diff
@@ -0,0 +1,1496 @@
+# Owner(s): ["oncall: quantization"]
+import copy
```

Comment:
Which changes have been made here to support `per_tensor` weight quantization as the default (instead of `per_channel`)?
If no changes are needed, could you elaborate as to why.